Configure

```diff
@@ -119,6 +119,9 @@ my $gcc_devteam_warn = "-DDEBUG_UNUSED"
         . " -Wswitch"
         . " -DPEDANTIC -pedantic -Wno-long-long"
         . " -Wall"
+        . " -Wextra"
+        . " -Wno-unused-parameter"
+        . " -Wno-missing-field-initializers"
```

Comment:
I've cross-examined this and spotted inconsistency. Configure gives clang a special treatment and it does it by *appending* clang-specific warning flags to gcc ones. This means that now there will be two -Wextras. [Well, right now clang will get pair of -Wextras anyway, so there will be *3* -Wextras.] Secondly, since it's appending, it will also duplicate -Wno-unsigned-parameters and -Wno-missing-field-initializers. In other words formally speaking correct line of action should be to *move* the above-mentioned options from $clang_devteam_warn, as opposed to *adding* them to $gcc_devteam_warn.

And while we are at it, it would be appropriate to remove -Wswitch and -Wundef duplicates. [It looks that one can also omit -Wno-parentheses-equality.] Redundant -Wextra will be addressed separately.